src/samples/Carousel/sass/Carousel.scss

```diff
@@ -0,0 +1,264 @@
+
+.ms-Carousel {
+  width: 100%;
+  position: relative;
+  height: $ms-Carousel-height;
+  background-color: $ms-Carousel-slideOneColor;
+  overflow: hidden;
+  transition: background-color 1000ms linear;
+  box-sizing: border-box;
+}
+
+.ms-Carousel-slide {
+  height: $ms-Carousel-height;
+  width: 100%;
+  max-width: $ms-Carousel-slideLargeWidth; 
+  margin-left: 0;
+  padding: 24px;
+  padding-top: 0;
+  box-sizing: border-box;
+  left: 50%;
+  transform: translateX(-50%);
+  position: absolute;
+
+  @media (min-width: $ms-screen-xl-min) {
+    top: 0;
+    padding: 0;
+  }
+}
+
+.ms-Carousel-innerSlide {
+  float: left;
+  position: relative;
+  height: 100%;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.ms-Carousel-slideTitle {
+  @include ms-font-xl;
+  color: $ms-color-white;
+  margin-bottom: 8px;
+  @media (min-width: $ms-screen-xl-min) {
+    @include ms-font-xxl;
+    color: $ms-color-white;
+  }
+}
+
+.ms-Carousel-slideSubTitle {
+  @include ms-font-m;
+  line-height: $ms-font-size-xl;
+  color: $ms-color-white;
+  @media (min-width: $ms-screen-xl-min) {
+    @include ms-font-l;
+    line-height: $ms-font-size-xxl;
+    color: $ms-color-white;
+  }
+}
+
+// Animation Layers
+.ms-Carousel-animationLayerOne,
+  .ms-Carousel-animationLayerTwo {
+    height: auto;
+    width: 100%;
+    position: relative;
+  @media (min-width: $ms-screen-xl-min) {
+    float: left;
+    width: 50%;
+    height: 100%;
+  }
+}
+
+.ms-Carousel-animationLayerOne.is-hidden {
+  opacity: 0;
+}
+
+.ms-Carousel-staticContent {
+  width: 100%;
+  max-width: $ms-Carousel-slideLargeWidth;
+  overflow: hidden;
+  z-index: $ms-zIndex-front;
+  height: auto;
+  box-sizing: border-box;
+  text-align: center;
+  h1 {
+    @include ms-font-su;
+    color: $ms-color-white;
+    padding-left: 0;
+    margin-left: 0;
+    margin-top: 38px;
+  }
+
+  @media (min-width: $ms-screen-xl-min) {
+    top: 0;
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    text-align: left;
+  }
+}
+
+.ms-Carousel-staticContentImage {
+  width: auto;
+  height: 50px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+  @media (min-width: $ms-screen-xl-min) {
+    width: auto;
+    height: auto;
+    margin-top: 0;
+  }
+}
+
+.ms-Carousel-controls {
+  width: 100%;
+  max-width: $ms-Carousel-slideLargeWidth;
+  overflow: hidden;
+  z-index: $ms-zIndex-front;
+  height: 50px;
+  box-sizing: border-box;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 16px;
+}
+
+.ms-Carousel-pagination {
+  height: auto;
+  width: auto;
+  position: absolute;
+  left: 50%;
+  height: 10px;
+  top: 50%;
+  transform: translateY(-50%) translateX(-50%);
+}
+
+.ms-Carousel-paginationButton {
+  color: $ms-color-white;
+  height: $ms-Carousel-paginationButtonSize;
+  font-size: $ms-Carousel-paginationButtonSize;
+  margin-right: 2px;
+  opacity: 0.5;
+}
+
+.ms-Carousel-paginationButton:hover {
+  cursor: pointer;
+}
+
+.ms-Carousel-paginationButton.is-active {
```

Comment:
Same as above--this could be nested under the parent paginationButton definition above